app/webpacker/components/Tickets/TicketWorkbenches/CompetitionResultActionerView/MiscActions/EventsMergedData/EventsMergedDataContent.jsx

```diff
@@ -0,0 +1,152 @@
+import { useMutation, useQuery, useQueryClient } from '@tanstack/react-query';
+import React, { useState } from 'react';
+import {
+  Button, Confirm, Message, Table,
+} from 'semantic-ui-react';
+import getEventsMergedData from '../../../../api/competitionResult/getEventsMergedData';
+import Loading from '../../../../../Requests/Loading';
+import Errored from '../../../../../Requests/Errored';
+import deleteResultsData from '../../../../api/competitionResult/deleteResultsData';
+
+export default function EventsMergedDataContent({ ticketDetails }) {
+  const queryClient = useQueryClient();
+  const [showConfirm, setShowConfirm] = useState();
+  const {
+    ticket: {
+      id: ticketId,
+      metadata: { competition_id: competitionId },
+    },
+  } = ticketDetails;
+
+  const {
+    data: eventsMergedData,
+    isPending: isEventsMergedDataPending,
+    isError: isEventsMergedDataError,
+    error: eventsMergedDataError,
+  } = useQuery({
+    queryKey: ['events-merged-data', ticketId],
+    queryFn: () => getEventsMergedData({ ticketId }),
+  });
+
+  const {
+    mutate: deleteResultsDataMutate,
+    isPending: isDeleteDataPending,
+    isError: isDeleteDataError,
+    error: deleteDataError,
+  } = useMutation({
+    mutationFn: deleteResultsData,
+    onSuccess: (_, { model, roundId }) => {
+      queryClient.setQueryData(
+        ['events-merged-data', ticketId],
+        (previousData) => previousData.map((roundData) => {
+          if (model === 'All') {
+            return {
+              ...roundData,
+              count: {
+                results: 0,
+                scrambles: 0,
+              },
+            };
+          }
+          if (roundData.round_id === roundId) {
+            return {
+              ...roundData,
+              count: {
+                ...roundData.count,
+                [model.toLowerCase()]: 0,
```

Comment:
Same here as above: Be careful about plural vs. singluar nouns.

Reply:
`model` is already singular here.